src/worldcereal/openeo/inference.py

```diff
@@ -0,0 +1,67 @@
+"""Model inference on Presto feature for binary classication"""
+
+import xarray as xr
+from openeo_gfmap.inference.model_inference import ModelInference
+
+
+class CroplandClassifier(ModelInference):
+    import functools
+
+    CATBOOST_PATH = "https://artifactory.vgt.vito.be/artifactory/auxdata-public/worldcereal-minimal-inference/wc_catboost.onnx"  # NOQA
```

Comment:
Same remark as with the feature extractor. This is the default model path, but we have to be able to set another one when creating an instance of this class.